dwl.c

```diff
@@ -892,8 +957,9 @@ pointerfocus(Client *c, struct wlr_surface *surface, double sx, double sy,
 		uint32_t time)
 {
 	/* Use top level surface if nothing more specific given */
-	if (c && !surface)
-		surface = c->xdg_surface->surface;
+	if (c && !surface) {
+		surface = WLR_SURFACE(c);
+	}
```

Comment:
It looks like this repo prefers to omit braces when an if contains a single statement. This change does not match that style.

Reply:
Fixed; thanks for spotting it.